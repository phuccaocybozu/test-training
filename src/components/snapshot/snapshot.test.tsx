import { render } from "@testing-library/react";
import SnapShot from ".";

describe("snapshot", () => {
  test("case 1", () => {
    const { container } = render(
      <SnapShot>This is content of Snapshot component 1</SnapShot>
    );

    expect(container).toMatchSnapshot();
  });

  test("case 2", () => {
    const { container } = render(
      <SnapShot>This is content of Snapshot component 2</SnapShot>
    );

    expect(container).toMatchSnapshot();
  });
});
