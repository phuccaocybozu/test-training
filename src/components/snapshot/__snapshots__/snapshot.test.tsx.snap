// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot case 1 1`] = `
<div>
  <div>
    <h2>
      Title
    </h2>
    <p>
      This is content of Snapshot component 1
    </p>
  </div>
</div>
`;

exports[`snapshot case 2 1`] = `
<div>
  <div>
    <h2>
      Title
    </h2>
    <p>
      This is content of Snapshot component 2
    </p>
  </div>
</div>
`;
